### ISSUE TITLE
Vikunja eating memory with large swap enabled on Raspberry Pi

### ISSUE THREAD

**Issue by DaCHack** [go-vikunja/vikunja#97]
### Description

Running Raspberry Pi OS on a RPi4 2GB

Vikunja runs fine under default settings with less than 5% memory usage.
When enabling 2GB of swap for another app I noticed that suddenly Vikunja took 40% of physical memory filling up the entire physical memory. Swap file was not used by the system yet due to swappiness at 0. Still, system got extremely slow so I need to revert back to default settings and disable swap to have all service available to the users again.

### Vikunja Frontend Version

0.21.0

### Vikunja API Version

0.21.0

### Browser and version

Edge

### Can you reproduce the bug on the Vikunja demo site?

No

**Comment by kolaente** [comment]
Sounds like a problem of the api. Were you doing anything during that time? Did it happen at a specific time? Anything in the logs when it happened?

**Comment by DaCHack** [comment]
No, I was not doing anything yet. I occured right after the reboot to enable swap and did not stop until I stopped the containers for vikunja. 

The logs show a couple of timeouts from database connections that I assume stem from the large latency when the system was memory flooded (while I do not get why it was slow despite no swap was used...).
Also this occurred as first output after boot which might have the same root cause since eventually vikunja was able to connect:

```
2023-11-27T21:57:04.02580685Z: CRITICAL	▶ migration/Migrate 002 Migration failed: dial tcp: lookup db on 127.0.0.11:53: no such host
2023-11-27T21:57:05.653843555Z: CRITICAL	▶ migration/Migrate 002 Migration failed: dial tcp 172.22.0.4:3306: connect: connection refused
2023-11-27T21:57:07.051648955Z: CRITICAL	▶ migration/Migrate 002 Migration failed: dial tcp 172.22.0.4:3306: connect: connection refused
2023-11-27T21:57:08.666376493Z: INFO	▶ migration/Migrate 052 Ran all migrations successfully.
2023-11-27T21:57:08.670324654Z: INFO	▶ cmd/func25 054 Vikunja version v0.21.0
```

**Comment by kolaente** [comment]
Does it always happen or only sometimes?

**Comment by DaCHack** [comment]
It happened on two consecutive boots

**Comment by kolaente** [comment]
Is this still reproducible?

**Comment by kolaente** [comment]
Closing as inactive, please ping or open a new issue with relevant information if you still have this problem.
